roles/test_sensubility/tasks/test_health_status.yml

```diff
@@ -1,8 +1,8 @@
   #In this test we choose one container (logrotate_crond),checking that it is running 
   #and then stopping the container and starting it again and checking that its health status changes accordingly.
 
-- name: RHELOSP-176011 Check that logrotate_crond container is running on controller-0
-  delegate_to: "{{ groups['overcloud_nodes'][0] }}"
+- name: RHELOSP-176011 Check that logrotate_crond container is running on ceph-0
```

Comment:
Is this container expected to run on only ceph? or all nodes?

Reply:
all